bootstrap.sh

```diff
@@ -85,6 +85,36 @@ if [[ ${master_settings_missing} == "true" ]]; then
   done
 fi
 
+# Create Elasticsearch snapshot repository
+if ! curl localhost:9200/_snapshot?pretty | grep $BACKUP_REPOSITORY; then
+
+  curl -XPUT localhost:9200/_snapshot/$BACKUP_REPOSITORY?pretty -H 'Content-Type: application/json' -d "{
+      \"type\": \"gcs\",
+      \"settings\": {
+          \"bucket\": \"${BACKUP_REPOSITORY}\",
+          \"base_path\": \"es_backup\"
+      }
+  }";
+fi
+
+# Create snapshot lifecycle management policy
+if ! curl localhost:9200/_slm/policy/?pretty | grep nightly-snapshots; then
+
+  curl -XPUT localhost:9200/_slm/policy/nightly-snapshots?pretty -H 'Content-Type: application/json' -d "{
```

Comment:
Is it possible to use some kind of heredoc? Escaping that many " could lead to typos

Reply:
Great idea! Fixed and looks much better now :+1: